### ISSUE TITLE
Question: SURVIVOR and SNIFFLE2

### ISSUE THREAD

**Issue by leone93** [fritzsedlazeck/Sniffles#368]
Hi, first of all thanks for the tools.
My question is related to the two tools.
Are the logic that both use for merging the SV call of different samples of a population the same? and also the default parameter to do that are the same? I want to understand if the results are comparable/the same merging the data with sniffle2 or with survivor.

**Comment by fritzsedlazeck** [comment]
Yeah thanks @leone93 for that question and sorry for the super late reply. 

So they are similar right now. This means that SURVIVOR just takes the wobble (1000bp) as is. Sniffles makes this a maximum but corrects it by the size of the SV. 

In the future, we plan to implement a more sensitive strategy comparable to Truvari. 
Thanks
Fritz
